exercises/palindrome-products/src/example.c

```diff
@@ -1,36 +1,109 @@
 #include "palindrome_products.h"
-#include <stdbool.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <limits.h>
 
-/* Return true if the product of a and b is a palindrome, false otherwise */
-bool is_palindrome_product(int a, int b)
+static int palindrome(int n);
+static int addfactors(factor_t ** p, int i, int k);
+static void free_ll(struct factors *p);
```

Comment:
I don't think there's any need to change, but something that helps to keep your code DRY (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is to avoid static prototypes by reorganizing your code so that helpers are always above the functions that use them. This means that if you ever refactor a name or a signature you only have to change it in one place, not two.